patterns/1-initial/principles.md

```diff
@@ -0,0 +1,152 @@
+## Title
+
+Explicit InnerSource Principles
+
+## Patlet
+
+InnerSource as a term is circulating among employees. The usual explanation of
+"applying open source best practices inside an organisation" does not work well
+though with people lacking an open source background. As a remedy the most
+important principles of InnerSource get published widely.
```

Comment:
```suggestion
The usual InnerSource explanation of "applying open source best practices 
inside an organisation" does not work well with people lacking an 
open source background. As a remedy the most important principles of 
InnerSource get published widely.
```

Shortening the Patlet by moving parts of it to Context.

The diff is unfortunately not that easy to read but I hope you can still see what I changed.